### ISSUE TITLE
Android firefox app

### ISSUE THREAD

**Issue by jukos** [andreaordonselli/qgis2opengis#7]
Hello,
with this browser app it is not possible to activate/check or deactivate/uncheck layers. 

Greetings

qgis2opengis 2.20
android 12
Firefox 126.0

**Comment by andreaordonselli** [comment]
What error does the console have? Have you made any changes to the code?

**Comment by jukos** [comment]
With your map - https://www.opengis.it/demo/ - the same. Also on my tablet with Android 11. Windows firefox is all ok.
I have no error news. 
Greetings

**Comment by jukos** [comment]
Excuse me, please. My error message is not correct.
Enable / disable works, but the boxes /check fields are hard to use in the firefox app compared to the chrome app.

**Comment by andreaordonselli** [comment]
Several years ago I spent a lot of time making Firefox read the system like other browsers do natively.  At a certain point I abandoned it, it doesn't update quickly enough, keep in mind that the css code still doesn't recognize the :has selector, while other browsers have already been doing so for 3 years.  If it's essential for you, I'll look into fixing it, otherwise there are chrome, edge and opera which work very well.  I hope you will understand my choice.
